docs/gcloud-auth.rst

```diff
@@ -0,0 +1,264 @@
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+
+Authentication
+--------------
+
+==============
+Quick overview
+==============
+
+**If you're running in Compute Engine or App Engine**,
+authentication should "just work"
+as credentials can be loaded automatically
+and therefore there is no extra code required to authenticate.
+
+**If you're developing locally**,
+the easiest way to authenticate is using the Cloud SDK.
+
+**If you're running your application elsewhere**,
+you should download the service account JSON keyfile
```

Comment:
s/the service account JSON keyfile/a service account JSON keyfile

Using "the" makes it sound like there is just one.